libbeat/reader/readfile/line_test.go

```diff
@@ -31,10 +35,12 @@ import (
 )
 
 // Sample texts are from http://www.columbia.edu/~kermit/utf8.html
-var tests = []struct {
+type LineTestSpec struct {
```

Comment:
Please do not export this struct as you are not using it from outside the package. Also, rather name it `listTestCase`. The name you have given it is not idiomatic to this repo.

Reply:
Updated